common/common-rest/src/main/java/io/servicecomb/common/rest/codec/param/RestClientRequestImpl.java

```diff
@@ -55,10 +76,20 @@ public Buffer getBodyBuffer() throws Exception {
     return bodyBuffer;
   }
 
+  @Override
+  public void attach(String name, String filename) {
+    uploads.put(name, filename);
+  }
+
   @Override
   public void end() throws Exception {
     writeCookies();
 
+    if (!uploads.isEmpty()) {
+      attachFiles();
```

Comment:
missed normal form parameter

io.servicecomb.common.rest.codec.param.RestClientRequestImpl.getBodyBuffer() invoke genBodyBuffer directly, there is conflict.